compiler/scopelang/scope_to_dcalc.ml

```diff
@@ -115,6 +115,45 @@ let tag_with_log_entry
         Pos.get_position e ))
     e
 
+(* In a list of exceptions, it is normally an error if more than a single one
+   apply at the same time. This relaxes this constraint slightly, allowing a
+   conflict if all the triggered conflicting exception yield syntactically equal
+   results (and as long as none of these exceptions have exceptions themselves)
+
+   NOTE: the choice of the exception that will be triggered and show in the
```

Comment:
I thought the encoding with a chain of exceptions made the choice of the triggered exception not abritrary any more ?

Reply:
well, I think that if we have `(a |- c)` and `(b |- c)`, the trace will show that condition `a` was triggered if we chain as `<(a |- c) | (b |- c)>`, or `b` if we chain as `<(b |- c) | (a |- c)>` ; both ways of chaining seeming equally correct ?

Reply:
The log operators are setup this way (you can show them in Dcalc with the `-d` flag) : `<( (print_trace_if_true a ) |- c) | ((print_trace_if_true b) |- c)>`.  So yes if both `a` and `b` are true then it will show `a` or `b` arbitrarily but deterministically, which is fine. But if only `a` or `b` is true then it will show the right one.